domain/src/test/java/jinsu/antilog/domain/ExpressionBufferTest.kt

```diff
@@ -0,0 +1,150 @@
+package jinsu.antilog.domain
+
+import com.google.common.truth.Truth.*
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+class ExpressionBufferTest {
+    @Test
+    fun `빈 수식에 피연산자를 추가하면 피연산자가 추가된 수식이 나온다`() {
+        // given : 빈 수식에
+        val expressionBuffer = ExpressionBuffer()
+        // when : 피연산자를 추가하면
+        val operand = Operand("5")
+        val actualExpression = expressionBuffer.addOperand(operand)
+        // then : 피연산자가 추가된 수식이 나온다.
+        assertThat(actualExpression).isEqualTo("5")
+    }
+
+    @Test
+    fun `피연산자와 연산자가 있는 수식에 피연산자를 추가하면 피연산자가 추가된 수식이 나온다`() {
+        // given : "5 + " 라는 수식에
+        val expressionBuffer = ExpressionBuffer().apply {
+            addOperand(Operand("5"))
+            addOperator(Operator.findOperatorBySymbol("+"))
```

Comment:
+를 찾는 로직을 테스트하는 것이 아니기 때문에, 프로덕션 로직 보다는 Operator.PLUS를 직접 하드코딩해서 넣어주는 것이 좋습니다.
```suggestion
            addOperator(Operator.PLUS)
```